streams/es.m3u

```diff
@@ -399,3 +457,27 @@ https://streaming007.gestec-video.com/hls/artequatreTVA.m3u8
 https://cloud3.streaminglivehd.com:1936/universfaller/universfaller/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="timeout",Urban Revolution
 https://www.urbanrevolution.es:444/live/5e6d8470a3832/index.m3u8
+#EXTINF:-1 tvg-id="" status="online",7 TeleValencia (576p)
+https://live-edge-ams-1.cdn.enetres.net/9E9557EFCEBB43A89CEC8FBD3C500247022/live-500/index.m3u8
```

Comment:
404. Not 24/7 ?

Reply:
It's possible, it was working around 10h ago.

Reply:
> 404. Not 24/7 ?

Actually, it is working right now:

![7TeleValencia](https://user-images.githubusercontent.com/33147829/210344708-1c255eb8-c12a-4126-8bbf-acb993f8f4fd.jpg)